### ISSUE TITLE
[Feature request] Distributed training

### ISSUE THREAD

**Issue by gronastech** [deepkit/deepkit#4]
Due to the fact that cluster entity already in place, can you please add Distributed training across all cluster nodes as described below:
https://lambdalabs.com/blog/tensorflow-2-0-tutorial-05-distributed-training-multi-node/

**Comment by marcj** [comment]
Yes, this is on our list already, but not on a high priority. We'll track the status in this issue.
